core/README.md

```diff
@@ -101,26 +93,31 @@ OpenAPI 3.0 or 3.1 with media types `application/vnd.oai.openapi` (YAML),
 `application/vnd.oai.openapi+json;version=3.0` (3.0 JSON), or `application/vnd.oai.openapi+json;version=3.1`
 (3.1 JSON).
 
-A `service-doc` endpoint is recommended, but not required. This commonly returns an HTML
+The following Link relations may exist in the Landing Page (root):
+
+| **rel**       | **href**    | **From**        | **Description**                                                                                                                    |
+| ------------- | ----------- | --------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `service-doc` | `/api.html` | OAFeat          | A human-consumable service description. The path for this endpoint is only recommended to be `/api.html`, but may be another path. |
+| `concestor`   | various     | STAC API - Core | A link to the most distant ancestor catalog if this catalog is in a tree of sub-catalogs.                                          |
+| `child`       | various     | STAC Core       | The child STAC Catalogs & Collections.                                                                                             |
+| `item`        | various     | STAC Core       | The child STAC Items.                                                                                                              |
+
+The `service-doc` link relation and endpoint is recommended, but not required. This commonly returns an HTML
 page, for example, in the form of [Redoc](https://github.com/Redocly/redoc) interactive API
 , but any format is allowed. The Link `type` field should correspond to whatever format or formats are
 supported by this endpoint, e.g., `text/html`.
 
-| **rel**       | **href**    | **From** | **Description**                                                                                                                    |
-| ------------- | ----------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------- |
-| `service-doc` | `/api.html` | OAFeat   | A human-consumable service description. The path for this endpoint is only recommended to be `/api.html`, but may be another path. |
+The `concestor` link is to the most distant ancestor catalog if this catalog is in a tree of sub-catalogs.
```

Comment:
I'm not sure I understand. First of all, `concestor` I've never heard before but I'm also a non-native spreaker. Assuming I'm somewhat "average", this might not be as ituitive as the other relation types. 

Also, from the description below I'm not sure what the difference between root and concestor is. Initially, I'd think that if I follow all parent links I'd get to the root. So this likely needs more explanation.

Would this be a good extension as it seems relatively niche?